src/components/Validator.vue

```diff
@@ -79,6 +87,8 @@ export default {
       memoryAlertMessage: '',
       durationAlertType: 'success',
       durationAlertMessage: '',
+      otherErrorAlertType: 'success',
+      otherErrorAlertMessage: '',
```

Comment:
"other" is ambiguous, I'd prefer to break it down to explicit types of errors.

Reply:
Fair enough, there are those three error types 'file format invalid', 'channel count invalid' and 'fseq version invalid'. If one of them is returned, then the other two errors will not be returned because each of them returns immediately from the validator. That's the reason I grouped them together, but I agree that the name 'other' isn't that well-chosen.